conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -110,8 +114,15 @@ private Contract createContract() {
     }
 
     private static Decoder createDecoder(ObjectMapper objectMapper, ObjectMapper cborObjectMapper) {
-        return new NeverReturnNullDecoder(new Java8OptionalAwareDecoder(new GuavaOptionalAwareDecoder(
-                new EmptyContainerDecoder(objectMapper, new InputStreamDelegateDecoder(new TextDelegateDecoder(
-                        new CborDelegateDecoder(cborObjectMapper, new JacksonDecoder(objectMapper))))))));
+        return new NeverReturnNullDecoder(
+                new Java8OptionalAwareDecoder(
+                        new GuavaOptionalAwareDecoder(
+                                new EmptyContainerDecoder(
+                                        objectMapper,
+                                        new InputStreamDelegateDecoder(
+                                                new TextDelegateDecoder(
+                                                        new CborDelegateDecoder(
+                                                                cborObjectMapper,
+                                                                new JacksonDecoder(objectMapper))))))));
```

Comment:
A bit extreme, but it's easier to read than the old code.

Reply:
amusingly I think this is exactly what we'd hand-rolled before the formatter was introduced!

Reply:
this is actually the fault of 0.3.17 (https://github.com/palantir/palantir-java-format/pull/157) and I think we can do something about it to bring the existing thing back...
or at least, something more similar to the existing thing

Reply:
I like the flying-v style here. Makes it clear that we should do something different instead of being dense and unreadable but out-of-the-way ;-)